src/server/services/afis/afis-facturen.ts

```diff
@@ -274,6 +277,10 @@ function transformFacturen(
   };
 }
 
+/** Checks if a factuur is available for download.
+ *
+ *  A factuur can only be downloaded after the day it is posted after 19:00 hours.
```

Comment:
```suggestion
 *  A factuur can only be downloaded after a certain time on the day it was posted. This because PDF's of invoices are generated by a batch job running at a later time.
```